website/docs/guides/getting_started.html.markdown

```diff
@@ -136,76 +136,51 @@ New Relic alerts are great, but they're even better when combined with good noti
 
 ```hcl
 # Notification channel
-resource "newrelic_alert_channel" "alert_notification_email" {
-  name = "username@example.com"
-  type = "email"
-
-  config {
-    recipients              = "username@example.com"
-    include_json_attachment = "1"
+resource "newrelic_notification_destination" "sample_notification_destination" {
+  name = "Sample Notification Destination"
+  type = "EMAIL"
+  
+  property {
+    key   = "email"
+    value = "username@example.com"
   }
 }
 
-# Link the above notification channel to your policy
-resource "newrelic_alert_policy_channel" "alert_policy_email" {
-  policy_id  = newrelic_alert_policy.alert_policy_name.id
-  channel_ids = [
-    newrelic_alert_channel.alert_notification_email.id
-  ]
-}
-```
-
-This example will send an email to the specified recipients whenever the associated alert condition is triggered. If you would like to send notifications via different modalities, such as Slack, the [alert channel](/providers/newrelic/newrelic/latest/docs/resources/alert_channel) resource supports multiple types of channels. Refer to the [additional alert channel examples](/providers/newrelic/newrelic/latest/docs/resources/alert_channel#type) for more examples in configuring your different notification channels.
-
-## A Note About Secrets
-
-As part of a `newrelic` resource, there is often some amount of configuration
-that is required in order for a resource to reach its full potential.  In some
-cases, once a given entity is created, API results will obscure the values for
-items that are deemed to be secret.  As a result, Terraform is unable to make
-an accurate detection of a resource state, and so marks a resource as changed
-for every run.
```

Comment:
Note for the reviewer: The changes discard this section, because
- This section briefly states that sensitive credentials are sometimes not returned by the API, so running `terraform plan` after `terraform apply` (after the resource is created) might result in a drift for a few credentials, so such attributes need to be added to `ignore_changes`. However, this no longer seems to be the case. This was the case when Terraform first introduced "sensitive credentials" and this discrepancy was seen, but no longer exists now. I tried running the following example and others in the provider which contain sensitive fields, but found a drift in none.
- Upon discussing all of the above with Sander, it was inferred that this section in the docs is outdated, owing to the above.